llvm/include/llvm/BinaryFormat/ELF.h

```diff
@@ -1939,11 +1939,12 @@ uint16_t convertArchNameToEMachine(StringRef Arch);
 /// Convert an ELF's e_machine value into an architecture name.
 StringRef convertEMachineToArchName(uint16_t EMachine);
 
-/// Convert a OS into ELF's EI_OSABI value.
-uint8_t convertOSToOSAbi(StringRef OS);
+// Convert a lowercase string identifier into an OSABI value.
+uint8_t convertNameToOSAbi(StringRef Name);
```

Comment:
Nit: whilst changing the signature, now might be a good time to fix the casing of `Abi` to `ABI`, since it's an acronym and therefore should be all uppercase. Same below, including the parameter.